index.html

```diff
@@ -9,8 +9,15 @@
 	</head>
 	<body>
 		<h1 id="text">Hello Git</h1>
-		<button type="button" onclick="changeColor()">Change color</button>
-
+		<div class="centered">
```

Comment:
Can we wrap the whole content in body with this `<div>` and set some width to have a nicer layout?